relayer/src/chain/counterparty.rs

```diff
@@ -505,30 +498,25 @@ pub fn acknowledgements_on_chain(
 pub fn unreceived_acknowledgements(
     chain: &impl ChainHandle,
     counterparty_chain: &impl ChainHandle,
-    channel: &IdentifiedChannelEnd,
-) -> Result<Vec<u64>, Error> {
-    let counterparty = channel.channel_end.counterparty();
-    let counterparty_channel_id = counterparty
-        .channel_id
-        .as_ref()
-        .ok_or_else(Error::missing_counterparty_channel_id)?;
-
-    let (commitments_on_src, _) =
-        commitments_on_chain(chain, &channel.port_id, &channel.channel_id)?;
+    path: &PathIdentifiers,
+) -> Result<(Vec<u64>, Height), Error> {
```

Comment:
Can we similarly update the doc comment for this method to reflect the new return type?